### DISCUSSION TITLE
Error 400 - Request too long on login

### DISCUSSION THREAD

**Discussion by frbenavide** [OrchardCMS/OrchardCore#15327] [Q&A] [answered]
Hi there guys!

Im getting this errror when trying to login into the OrchardCMS site.

Thanks in advance for the help.

![image](https://github.com/OrchardCMS/OrchardCore/assets/159968633/c0ade1ad-31b3-462a-8fa0-ac3ac51aa007)

**Comment by ns8482e** [comment, marked as answer]
Most likely due to larger cookie - #13142. Solution is to enable Users Authentication Ticket Store feature 

https://docs.orchardcore.net/en/main/docs/reference/modules/Users/TicketStore/#users-authentication-ticket-store
